source/mongorestore.txt

```diff
@@ -864,6 +864,13 @@ Options
    specified in the corresponding :binary:`~bin.mongodump` output.
 
 
+.. option:: --fixDottedHashIndex 
+
+   Creates all :manual:`hashed indexes 
+   </core/indexes/index-types/index-hashed/>` on dotted fields as 
+   single field ascending indexes on the destination database if
+   enabled. 
```

Comment:
It looks like the other documented options don't use "if/when enabled" language, so suggest removing for consistency.

```suggestion
   single field ascending indexes on the destination database.
```